buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -147,6 +150,34 @@ class ClusterFormationTasks {
                     into node.baseDir
                 }
                 break;
+            case 'rpm':
+                File rpmDatabase = new File(node.baseDir, 'rpm-database')
+                File rpmExtracted = new File(node.baseDir, 'rpm-extracted')
+                extract = project.tasks.create(name: name, type: Exec, dependsOn: extractDependsOn) {
+                    commandLine 'rpm', '--badreloc', '--nodeps', '--noscripts', '--notriggers',
+                        '--dbpath', rpmDatabase,
+                        '--relocate', "/=${rpmExtracted}",
+                        '-i', project.configurations.elasticsearchDistro.singleFile
+                    standardOutput = new ByteArrayOutputStream()
+                    errorOutput = standardOutput
+                    /* rpm complains about the database being corrupted and exits. But it gets far
```

Comment:
It complains about the database directory that we feed it. Its always complained about it in maven too, so far as I can tell. The vagrant tests are the "real" rpm tests for that kind of thing.